apps/test-bundles/webpack.config.js

```diff
@@ -12,7 +30,6 @@ const entries = buildEntries('@fluentui/react');
 //   true /* onlyOwnComponents */,
 // );
 
-// Create entries for single top level import.
 entries['react-compose'] = buildEntry('@fluentui/react-compose');
```

Comment:
Are these left as-is because we're testing compose and keyboard-key with both package groups?

Reply:
Removed the comment as it is not useful and it will be combined with React pipeline